packages/jest-circus/src/types.ts

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+type Process = NodeJS.Process; // eslint-disable-line no-undef
+
+export type DoneFn = (reason?: string | Error) => void;
+export type BlockFn = () => void;
+export type BlockName = string;
+export type BlockMode = void | 'skip' | 'only' | 'todo';
+export type TestMode = BlockMode;
+export type TestName = string;
+export type TestFn = (done?: DoneFn) => Promise<any> | null | undefined;
```

Comment:
not `null`, only `undefined`

Reply:
Only for `TestFn` or others too?

Reply:
There are many places where the values are assigned to `null`

Reply:
I guess it should be

```ts
export type TestFn = (done?: DoneFn) => Promise<any> | void | undefined;
```

>There are many places where the values are assigned to `null`

Really? That's a runtime error:

https://github.com/facebook/jest/blob/7ae3f076a03e24e570857db52ce43df263d22f67/packages/jest-circus/src/utils.js#L211-L220

Reply:
My bad, `null` s were assigned other types